locales/en-US/App.json

```diff
@@ -232,5 +232,7 @@
   "into": "into",
   "out from": "out from",
   "Transaction details": "Transaction details",
-  "Copy address": "Copy address"
+  "Copy address": "Copy address",
+  "Unlocks at": "Unlocks at",
+  "Unlocked at": "Unlocked at"
```

Comment:
A couple more that we need in French :) @mvaivre 

I think that once your #371 is merged, we should include both translations in every PR that changes something in English.

Reply:
As discussed, will happen in the other branch #371